data/human/variants.txt

```diff
@@ -97,6 +97,69 @@ ship "Arrow" "Arrow (Hai)"
 
 
 
+ship "Auxiliary" "Auxiliary (Mark II)"
+	outfits
+		"Sidewinder Missile Launcher" 2
+		"Sidewinder Missile" 90
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret" 2
+		
+		"Fusion Reactor"
+		"LP288a Battery Pack"
+		"D94-YV Shield Generator"
+		"D67-TM Shield Generator"
+		"Water Coolant System"
+		"Large Radar Jammer"
+		"Catalytic Ramscoop"
+		
+		"X4700 Ion Thruster"
+		"X1700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Scram Drive"
+
+
+ship "Auxiliary (Cargo)" "Auxiliary (Cargo Mark II)"
```

Comment:
The game doesn't like variants of variants. You'll have to partially repeat the Auxiliary (Cargo) variant definition.

Reply:
Thank you, i was trying to figure out why the check failed! Would it be best to separate it like this: 
```suggestion
ship "Auxiliary (Cargo)" "Auxiliary (Cargo) (Mark II)"
```
Or some other way? Or do you mean i'd have to repeat something else?

Reply:
Tried it and it seems happy... alright.

Reply:
Doing that will cause the model name of the Auxiliary to show up as "Auxiliary (Cargo)". You have to do something like this instead:

https://github.com/endless-sky/endless-sky/blob/a0fa9c54d17c7a868102204e512936478ba87039/data/human/variants.txt#L100-L135

Reply:
Ah, i see, i need to add the atributes like the variants do so on ships.txt on variants.txt as well. Thank you so much for the help!